plugins/ScheduledReports/API.php

```diff
@@ -86,18 +87,22 @@ public function __construct(LoggerInterface $logger)
      * @param array $reports array of reports
      * @param array $parameters array of parameters
      * @param bool|int $idSegment Segment Identifier
+     * @param bool $evolutionPeriodFor If true, evolution graphs cover each day within the period. If false, evolution graphs
```

Comment:
I'm actually not quite understanding the difference between the two but reckon will get more clear once having a look at it.

Reply:
`$evolutionPeriodFor` determines if looking within range or using lastN value. I originally only had one parameter `$evolutionPeriodLastN` where if it was <= 0, it would look at days within the range, but that's not really clear w/ just one parameter, so I opted for being explicit.